zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -41,12 +41,13 @@ <h3 class="new_h3" style="float:left">
       <div class="col-md-12">
         <h3 class="interpreter-title">{{setting.name}}
           <small>
-            <span ng-repeat="interpreter in setting.interpreterGroup"
+            <span style="display:inline-block" ng-repeat="interpreter in setting.interpreterGroup"
                   title="{{interpreter.class}}">
-              <span ng-show="$parent.$first && $first">%{{setting.name}} (default)</span>
-              <span ng-show="$parent.$first && !$first">, %{{interpreter.name}}</span>
-              <span ng-show="!$parent.$first && $first">%{{setting.name}}</span>
-              <span ng-show="!$parent.$first && !$first">, %{{setting.name}}.{{interpreter.name}}</span>
+              <span ng-show="!$first">, </span>
+              %<span ng-show="!$parent.$first || $first">{{setting.group}}</span
```

Comment:
I understand, I guess I'm not completely sure about the logical expression here.
If we want to show the group name for all group and interpreters that are not the first group, shouldn't we simply check `%<span ng-show="!$parent.$first>` (my previous comment was wrong)?